test/functional/cmdLineTests/gcRegressionTests/src/com/ibm/tests/garbagecollector/TestContractionMain.java

```diff
@@ -67,12 +67,8 @@ public static void main(String[] args)
 			long highWaterMem = runtime.totalMemory();
 			_array = null;
 			System.gc();
-			System.gc();
-			System.gc();
```

Comment:
Talked with @dmitripivkine , 3 GC in the row are needed here. Sorry for misleading. Please rever this change. Same for L74-75.

Reply:
Okay, thank you. I have reverted the three GC calls again .